src/app/map/index.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  GoogleMap,
+  Marker,
+  useLoadScript,
+  InfoWindow,
+} from "@react-google-maps/api";
+import { useEffect, useState } from "react";
+
+const Map = () => {
```

Comment:
@sweiyang , actually i want to have a props interface address, lat, lng. Let the ev-charging points pass from parent component. 

That will be more correct right?

Reply:
yea, it will be better. as the map is just a module that takes in a list of locations. if its troublesome i think we can just think about it later.

Reply:
see if you wanna do it in this PR or some other PR. If not can resolve this comment :)